resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -42,6 +43,7 @@
     private DnsCache resolveCache;
     private DnsCnameCache cnameCache;
     private AuthoritativeDnsServerCache authoritativeDnsServerCache;
+    private InetSocketAddress localAddress;
```

Comment:
```suggestion
    private SocketAddress localAddress;
```

Reply:
Sorry, I don't know I can commit the suggestion, I Fixed manually